web_src/js/features/comp/ComboMarkdownEditor.js

```diff
@@ -315,3 +310,9 @@ export async function initComboMarkdownEditor(container, options = {}) {
   await editor.init();
   return editor;
 }
+
+export function removeLinksInTextarea(editor, file) {
+  const fileName = file.name.slice(0, file.name.lastIndexOf('.'));
+  const fileText = `\\[${fileName}\\]\\(/attachments/${file.uuid}\\)`;
+  editor.value(editor.value().replace(new RegExp(`<img [\\s\\w"=]+ alt="${fileName}" src="/attachments/${file.uuid}">`, 'g'), '').replace(new RegExp(`\\!${fileText}`, 'g'), '').replace(new RegExp(fileText, 'g'), ''));
+}
```

Comment:
Use https://github.com/sindresorhus/escape-string-regexp here, later to be replaced with https://github.com/tc39/proposal-regex-escaping.

Reply:
IMO this function is not needed, why it should replace HTML `<img>` syntax to markdown syntax.